File: JavaScript/1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

function highest(a, b){
    return (a + b + Math.abs(a-b))/2
}

var line = lines.shift();
line = line.split(' ')

var a = parseInt(line[0])
var b = parseInt(line[1])
var c = parseInt(line[2])

x = parseInt(highest(a, b))
y = parseInt(highest(x, c))

console.log(y + " eh o maior")

File: C/1013.c
#include <stdio.h>
#include <stdlib.h>

int highest(int a, int b){
    return (a + b + abs(a-b))/2;
}

int main() { 
    int a, b, c, x, y;
    
    scanf("%d %d %d", &a, &b, &c);

    x = highest(a, b);
    y = highest(x, c);

    printf("%d eh o maior\n", y);

    return 0;
}